src/ProcessApi/DSCode.java

```diff
@@ -0,0 +1,15 @@
+package ProcessApi;
```

Comment:
change the package name to processapi. the naming convention in check style rules it